src/other/Table.js

```diff
@@ -557,8 +603,41 @@
         HTMLWidget.prototype.exit.apply(this, arguments);
     };
         
+    Table.prototype.setColumnWidths = function() {
+        var context = this;
+        var tableMarginHeight = this.fixedHeader() ? this.thead.node().offsetHeight : 0;
+
+        var totalWidth = 0;
+        var tds = this.table.select("tbody").selectAll("tr").selectAll("td");
```

Comment:
This should have been `select("tr")` instead of `selectAll` as per my comment above, thanks for catching.  It is interesting to note however, it does still work.  This selection is used to match the size of the body cells to their corresponding header cells.  The code below takes this selection and gets the size of the cells, compares them, and uses the largest value as the size for both (header and body).  Since a child table can't really have a `td` with a larger size than the cell that's containing it, its value will never be used.  That said, its a mute point because "select" is the correct method here.